Resources/Textures/_Moffstation/PersonalItems/Items/Chartman95/Peedee/peedeeavatar.rsi/meta.json

```
@@ -1,5 +1,5 @@
 ﻿{
-    "version": 1,
+    "version": 1.1,
```

Test fail here, it needs to be an integer. Just say it's version 2.

---

```suggestion
    "version": 1,
```
`version` here refers to the RSI format's version, not to a version of the content. It must be `1`.

https://docs.spacestation14.com/en/specifications/robust-station-image.html#:~:text=A%20simple%20integer%20corresponding%20to%20the%20RSI%20format%20version.%20This%20can%20be%20used%20to%20identify%20what%20version%20an%20RSI%20is%20and%20allow%20the%20implementation%20to%20correctly%20enable%20backwards%20compatibility%20modes%20when%20needed.

---

Yo Eres stupido